src/main/java/com/marianbastiurea/lifeofbees/Hive.java

```diff
@@ -401,12 +402,11 @@ public void beesDie(Date currentDate) {
 
     public boolean checkIfAll6EggsFrameAre80PercentFull() {
```

Comment:
This method looks too complicated or wrong. Fix or simplify.

Reply:
done